railties/test/application/configuration_test.rb

```diff
@@ -1719,5 +1737,10 @@ def index
       assert_equal 301, last_response.status
       assert_equal "https://example.org/", last_response.location
     end
+
+    private
+      def lazy_load
```

Comment:
I think the name of this method is not correct. We are not lazy loading the content of the block, we are forcing the lazy loads hooks to run. So better to call it `force_lazy_load_hooks`.